### DISCUSSION TITLE
Color display in the status bar

### DISCUSSION THREAD

**Discussion by og900aero** [gokcehan/lf#1069] [Q&A] [answered]
It's okay to write anything in the status line as follows: 
lf -remote "send echo 'Operation completed'"

But how can you write something in color? So, for example, how can I write with red text: Operation is completed?

**Comment by joelim-work** [comment, marked as answer]
You have to use escape sequences, just like what you would use for the `promptfmt` option:

```shell
lf -remote 'send echo "\033[31mOperation completed\033[0m"'
```

**Reply by joelim-work** [reply]
Hi @og900aero, hopefully the above suggestion works for youu, if so then please mark it as answered. This makes it easier for contributors to filter for unanswered discussions.
